web/packages/shared/components/TraitsEditor/TraitsEditor.tsx

```diff
@@ -257,6 +263,7 @@ export type TraitEditorProps = {
   configuredTraits: TraitsOption[];
   isLoading: boolean;
   tooltipContent?: React.ReactNode;
+  label?: string;
```

Comment:
add comment? we actually have something verrry similar to this component called the `LabelsInput.tsx`, so I got confused when i saw `label`